scripts/missions/rotz/05_Headstone_Pilgrimage.lua

```diff
@@ -96,8 +96,7 @@ mission.sections =
                     if option == 1 then
                         player:messageSpecial(behemothsDominionID.text.AIR_AROUND_YOU_CHANGED)
 
-                        SpawnMob(behemothsDominionID.mob.ANCIENT_WEAPON)
-                        SpawnMob(behemothsDominionID.mob.LEGENDARY_WEAPON)
+                        SpawnMob(behemothsDominionID.mob.ANCIENT_WEAPON):updateClaim(player)
```

Comment:
Curious about this change since I did it originally. Do you have anything showing that the Ancient Weapon should aggro to the player on pop? I know wiki isn't super reliable but it does say you should be able to pop them with sneak on and they don't aggro.

<img width="803" alt="image" src="https://user-images.githubusercontent.com/48370698/202524375-2b18ecbc-4b1d-4bad-a453-bb1f7f373100.png">

Reply:
I believe the bug reporter linked a video in the report itself of the other weapon ignoring the players while they killed the ancient weapon

Reply:
It should NOT aggro the player on pop. I tested it on retail for LB